### ISSUE TITLE
🐛 BUG: Cloudflare adapter 0.1.0-alpha.95 - /assets/entry-* - "404 Not Found" when run.

### ISSUE THREAD

**Issue by davycoe** [solidjs/solid-start#211]
Versions:

```
OS: Linux
node: v16.16.0
yarn: 1.22.19
solid-start: 0.1.0-alpha.95
solid-start-node: 0.1.0-alpha.95
solid-start-cloudflare-pages: 0.1.0-alpha.95
wrangler: 2.0.23
vite: 3.0.2
```

Hello, when running wrangler the the following assets result in a  404 Not Found: 
```
GET /assets/entry-client.022d2253.css 404 Not Found (9.15ms)
GET /assets/entry-client.e02fc6fa.js 404 Not Found (4.39ms)
```

1. yarn build (`"build": "solid-start build"`)
2. yarn start (`"wrangler pages dev dist/"`)

vite.config.ts:
```
/* eslint-disable */

import { defineConfig } from "vite";
import tsconfigPaths from "vite-tsconfig-paths";
import solid from "solid-start";
//@ts-ignore
import cf from "solid-start-cloudflare-pages";

export default defineConfig({
  plugins: [solid({ adapter: cf() }), tsconfigPaths()],
});
```

Output:
```
yarn start
yarn run v1.22.19
warning package.json: No license field
$ wrangler pages dev dist/
🚧 'wrangler pages <command>' is a beta command. Please report any issues to https://github.com/cloudflare/wrangler2/issues/new/choose
Compiling worker to "/tmp/functionsWorker-0.9209805462891729.js"...
Compiled Worker successfully.
╭─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮
│ [b] open a browser, [d] open Devtools, [c] clear console, [x] to exit                                               │
╰─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
Starting inspector on localhost:9229 failed: address already in use
[pages:wrn] Service bindings are experimental. There may be breaking changes in the future.
[pages:inf] Worker reloaded! (76.88KiB)
[pages:inf] Listening on localhost:8788
[pages:inf] - http://localhost:8788
[pages:inf] Updated `Request.cf` object cache!
GET / 200 OK (49.04ms)
GET /assets/entry-client.022d2253.css 404 Not Found (9.15ms)
GET /assets/entry-client.e02fc6fa.js 404 Not Found (4.39ms)

```

Also separate issue, when updating to `vite 3.0.4` during the build script it blows up with/without an adapter:
```
$ solid-start build                                                                                                    
vite v3.0.4 building for production...                                                                                 
✓ 27 modules transformed.                                                                                              
dist/public/manifest.json                      0.77 KiB                                                                
dist/public/ssr-manifest.json                  1.06 KiB                                                                
dist/public/assets/_...404_.55eb87fa.js        0.39 KiB / gzip: 0.30 KiB                                               
dist/public/assets/index.733a17c0.js           0.96 KiB / gzip: 0.56 KiB                                               
dist/public/assets/entry-client.022d2253.css   5.76 KiB / gzip: 1.56 KiB                                               
dist/public/assets/entry-client.e02fc6fa.js    32.63 KiB / gzip: 12.36 KiB                                             
vite v3.0.4 building SSR bundle for production...                                                                      
✓ 37 modules transformed.                                                                                              
.solid/server/manifest.json                 0.64 KiB                                                                   
.solid/server/entry-server.js               45.37 KiB                                                                  
.solid/server/assets/_...404_.c3f82c50.js   0.49 KiB                                                                   
.solid/server/assets/index.375dd778.js      1.13 KiB                                                                   
file:///home/coe/code/solid-starter/node_modules/rollup/dist/es/shared/rollup.js:1858                                  
        base = Object.assign(new Error(base.message), base);                                                           
                             ^                                                                                         

Error: Invalid value for option "output.file" - when building multiple chunks, the "output                             .dir" option must be used, not "output.file". To inline dynamic imports, set the "inlineDy                             namicImports" option.                                                                                                  
    at error (file:///home/coe/code/solid-starter/node_modules/rollup/dist/es/shared/rollu                             p.js:1858:30)                                                                                                          
    at validateOptionsForMultiChunkOutput (file:///home/coe/code/solid-starter/node_module                             s/rollup/dist/es/shared/rollup.js:16124:16)                                                                            
    at Bundle.generate (file:///home/coe/code/solid-starter/node_modules/rollup/dist/es/sh                             ared/rollup.js:15956:17)                                                                                               
    at async file:///home/coe/code/solid-starter/node_modules/rollup/dist/es/shared/rollup                             .js:23709:27                                                                                                           
    at async catchUnfinishedHookActions (file:///home/coe/code/solid-starter/node_modules/                             rollup/dist/es/shared/rollup.js:23041:20)                                                                              
    at async Object.build (file:///home/coe/code/solid-starter/node_modules/solid-start-cl                             oudflare-pages/index.js:70:7) {                                                                                        
  code: 'INVALID_OPTION',                                                                                              
  url: 'https://rollupjs.org/guide/en/#outputdir'                                                                      
}                                                                                               
error Command failed with exit code 1.                                                                                 
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command. 
```

**Comment by nksaraf** [comment]
1. For the assets, after version 0.1.0-alpha.95, we started to use the `dist/public` folder for static assets, so you should pass that to the `wrangler` script, eg.

```
wrangler pages dev ./dist/public
```

For the build step issue, I think the new vite/rollup versions changed the defaults. This is fixed in 0.1.0-alpha.97 (not released yet) (uses commit 56e91a6e9502eea0731d10ba362f35e062cf2dca) where we make sure to `inlineDynamicImports` in the server build.

**Comment by davycoe** [comment]
Thank you, don't know how I overlooked the change to `dist/public` haha!
